sheetstorm/management/commands/generate_test_data.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+CustomUser = get_user_model()
+
+superuser_user_type = "SUPERUSER"
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+
+        user_options = self._get_user_options(options)
+        is_need_to_create_superuser = self._get_superuser_request(options)
+
+        for (user_type, number_of_users) in user_options.items():
+            if number_of_users is not None:
+                self.add_user(user_type, number_of_users)
+
+        if is_need_to_create_superuser:
+            self.add_user(superuser_user_type)
+
+        logging.info(f"Total number of users in the database: {CustomUser.objects.count()}")
+
+    @staticmethod
+    def _get_user_options(options):
+
+        user_options = {
+            CustomUser.UserType.ADMIN.name: options[CustomUser.UserType.ADMIN.name],
+            CustomUser.UserType.EMPLOYEE.name: options[CustomUser.UserType.EMPLOYEE.name],
+            CustomUser.UserType.MANAGER.name: options[CustomUser.UserType.MANAGER.name],
+        }
+        return user_options
```

Comment:
Please simplify it to:
```python
return {
            CustomUser.UserType.ADMIN.name: options[CustomUser.UserType.ADMIN.name],
            CustomUser.UserType.EMPLOYEE.name: options[CustomUser.UserType.EMPLOYEE.name],
            CustomUser.UserType.MANAGER.name: options[CustomUser.UserType.MANAGER.name],
        }
```

When you are creating just one variable there is no need to add it separately. You can just return it